src/lib/reasoners/adt_rel.ml

```diff
@@ -319,45 +319,9 @@ let empty uf = {
 (* ################################################################ *)
 (*BISECT-IGNORE-BEGIN*)
 module Debug = struct
-  open Printer
-
-  let assume a =
-    if Options.get_debug_adt () then
-      print_dbg
-        ~module_name:"Adt_rel"
-        ~function_name:"assume"
-        "assume %a" LR.print (LR.make a)
-
-  let pp_domains loc domains =
-    if Options.get_debug_adt () then begin
-      print_dbg ~flushed:false ~module_name:"Adt_rel"
-        "@[<v 2>--ADT env %s ---------------------------------@ " loc;
```

Comment:
Looks like this PR removes the the `---` separators that can make the logs more readable; is that intentional?

Reply:
It is intentional because `Logs` uses correctly indentation. So we will get something like that:
```
[DEBUG][Adt_rel] environment before assume:
                             the environment here 
                             .... 
[DEBUG][Adt_rel] ... 
``` 
The indentation makes clear in which messages we are.